src/FileDropper/FileDropper.stories.tsx

```diff
@@ -0,0 +1,113 @@
+import { Button } from '@/buttons';
+import { TABLE_CATEGORIES } from '@/utils/storybook';
+import { expect } from '@storybook/jest';
+import type { Meta, StoryObj } from '@storybook/react';
+import { within } from '@storybook/testing-library';
+
+import { Box } from '@mui/material';
+
+import FileDropper from './FileDropper';
+
+const meta = {
+  title: 'Common/FileDropper',
+  component: FileDropper,
+
+  argTypes: {
+    onChange: {
+      table: {
+        category: TABLE_CATEGORIES.EVENTS,
+      },
+      action: 'on change',
+    },
+    onDrop: {
+      table: {
+        category: TABLE_CATEGORIES.EVENTS,
+      },
+      action: 'on drop',
+    },
+  },
+} satisfies Meta<typeof FileDropper>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const Default: Story = {
+  args: {
+    onChange: () => {
+      console.log('on change');
+    },
+    onDrop: () => {
+      console.log('woiejfkl');
+    },
+  },
```

Comment:
You can set default arguments in the meta, for functions use `fn()` that you import from `import { fn } from '@storybook/test';`. This is useful so tests do not complain.